pkg/controller/siddhiprocess/operator.go

```diff
@@ -124,7 +124,7 @@ func (rsp *ReconcileSiddhiProcess) deployApp(
 			RollingUpdate: &rollingUpdate,
 		}
 	}
-	configMapName := siddhiApp.Name + strconv.Itoa(int(sp.ObjectMeta.Generation))
+	configMapName := siddhiApp.Name + strconv.Itoa(int(sp.Status.CurrentVersion))
```

Comment:
You can use `fmt.Sprintf` instead of converting int to string

Reply:
Fixed in https://github.com/siddhi-io/siddhi-operator/pull/123/commits/2771b68508ad07644397209e0c2107758f286b90